packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.js

```diff
@@ -26,6 +26,12 @@ const Tool = types
     /*
     Create tasks from details if we have details but no tasks.
     */
+    if (!snapshot.max) {
```

Comment:
[minor][rambling] Sanity check: this implies that a Drawing Tool can never have "maximum number of marks = 0". Response: that's legit.

Reply:
[minor][rambling] I just realised it's theoretically possible to have odd logic like max and min being negative numbers, or max < min, but those logic checks aren't actually necessary since if a drawing tool is _that_ borked, the Project Owner would know pretty darn quickly.